src/GitHub.VisualStudio/Views/Dialog/RepositoryCloneView.xaml

```diff
@@ -166,13 +166,14 @@
                 HorizontalAlignment="Center"
                 VerticalAlignment="Center"
                 DockPanel.Dock="Top">
-      <uirx:ErrorMessageDisplay x:Name="loadingFailedMessage"
-                                Margin="8,0"
-                                Content="{x:Static prop:Resources.loadingFailedMessageContent}"
-                                Icon="stop"
-                                Message="{x:Static prop:Resources.loadingFailedMessageMessage}"
-                                Visibility="{Binding LoadingFailed, Mode=OneWay, Converter={ui:BooleanToVisibilityConverter}}" />
-    </StackPanel>
+        <uirx:ErrorMessageDisplay x:Name="loadingFailedMessage"
+                                  Margin="8,0"
+                                  Icon="stop"
+                                  Message="{x:Static prop:Resources.loadingFailedMessageMessage}"
+                                  Visibility="{Binding LoadingFailed, Mode=OneWay, Converter={ui:BooleanToVisibilityConverter}}">
+          <TextBlock Text="{x:Static prop:Resources.loadingFailedMessageContent}" TextWrapping="Wrap" />
+        </uirx:ErrorMessageDisplay>  
+        </StackPanel>
```

Comment:
Nit: Looks like the indentation got messed up here - you need to be careful when editing XAML files - some of our XAML files use a tab spacing of 4 and some 2 (and some are mixed, arrgh!) and VS will go ahead and reformat things for you if you have the wrong setting! It's very annoying, we should fix it, but in the meantime probably best to manually fix this indentation.

Reply:
thanks @grokys will update 👍